src/narrowphase/narrowphase.cpp

```diff
@@ -2555,12 +2555,46 @@ bool planeIntersect(const Plane& s1, const Transform3f& tf1,
 
 } // details
 
+// Shape intersect algorithms not using libccd
+//
+// +------------+-----+-----------+--------+---------+------+----------+-------+------------+----------+
+// |            | box | ellipsoid | sphere | capsule | cone | cylinder | plane | half-space | triangle |
```

Comment:
ellipsoids aren't in the master branch yet, though they are in #51 

I'm ok with them having a slot in this table though, to save on editing the ascii art